scripts/ngless-count.py

```diff
@@ -20,80 +21,48 @@ def parse_args():
                         help="SAM/BAM/CRAM file to count reads on")
     parser.add_argument("-o", "--output", required=True,
                         help="Output file/path for results")
+    parser.add_argument("-f", "--features",
+                        choices=["seqname", "gene"],
```

Comment:
I'm not sure how `choices` works, but the features is open world (ie, could be anything). We use it, for example, for specI profiling, by having a GFF file where features is "specI".